runelite-client/src/main/java/net/runelite/client/ui/ContainableFrame.java

```diff
@@ -27,23 +27,40 @@
 import java.awt.Frame;
 import java.awt.Rectangle;
 import javax.swing.JFrame;
+import lombok.RequiredArgsConstructor;
 import lombok.Setter;
 import net.runelite.client.config.ExpandResizeType;
 
 public class ContainableFrame extends JFrame
 {
+	@RequiredArgsConstructor
+	public enum Mode
+	{
+		ALWAYS("Always"),
+		RESIZING("Resizing"),
+		NEVER("Never");
+
+		public final String name;
```

Comment:
oh, actually. you do not need to do this anymore, config manager is handling simple naming like this now, so remove both name and toString